boards/nxp/mimxrt1170_evk/Kconfig.sysbuild

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+choice MCUBOOT_MODE
+	default MCUBOOT_MODE_SWAP_WITHOUT_SCRATCH if BOARD_MIMXRT1170_EVK_MIMXRT1176_CM7
```

Comment:
MCUBOOT_MODE_SWAP_WITHOUT_SCRATCH is already a default mode.
So no need to duplicate it here.

Reply:
From sysbuild? If you mean from a mcuboot conf file then it will be overwritten by sysbuild's value

Reply:
Adding of the boards/nxp/mimxrt1170_evk/Kconfig.sysbuild is not required.